Fix bug with returning to the app a second time

sestevens opened the pull request:
Fix a bug where returning from `SFSafariViewController` only worked the first time by not setting the returnBlock to nil.

billwerges commented on PopupBridge/Classes/POPPopupBridge.m:
```diff
@@ -55,7 +55,7 @@ - (id)initWithWebView:(WKWebView *)webView delegate:(id<POPPopupBridgeDelegate>)
                 NSMutableDictionary *payloadDictionary = [NSMutableDictionary new];
                 payloadDictionary[@"path"] = path;
-                payloadDictionary[@"queryItems"] = [self.class dictionaryForQueryString:url.query];
+                payloadDictionary[@"queryItems"] = [POPPopupBridge dictionaryForQueryString:url.query];
```
These instances of `self.class` should be changed to `weakSelf.class`. In general we want to avoid using hard coded class names because it can break sub-classing by preventing method overriding.

sestevens replied:
We figured it was OK to use the class name in this case since these are private methods that can't be overridden in a subclass. But I'm happy to change it to `weakSelf.class` if you prefer.

sestevens replied:
Updated ✔️

scannillo commented:
Looks good + CHANGELOG entry

sestevens commented:
> Looks good + CHANGELOG entry

Good catch! I added a changelog entry. I also added a PR template to #24 so that we'll have a reminder about the changelog in the future 🙂
